tests/foreman/ui_airgun/test_discoveredhost.py

```diff
@@ -0,0 +1,496 @@
+@tier3
+def test_positive_delete(session, module_org, module_loc, provisioning_env):
+    """Delete the selected discovered host
+
+    :id: 25a2a3ea-9659-4bdb-8631-c4dd19766014
+
+    :Setup: Host should already be discovered
+
+    :expectedresults: Selected host should be removed successfully
+
+    :CaseLevel: System
+    """
+    with LibvirtGuest() as pxe_host:
+        host_name = pxe_host.guest_name
+        with session:
+            session.organization.select(org_name=module_org.name)
+            session.location.select(loc_name=module_loc.name)
+            discovered_host_values = session.discoveredhosts.wait_for_entity(
+                host_name)
+            assert discovered_host_values['Name'] == host_name
+            session.discoveredhosts.delete(host_name)
+            assert not session.discoveredhosts.search(
+                'name = {0}'.format(host_name))
+
+
+@run_in_one_thread
+@run_only_on('sat')
+@skip_if_not_set('compute_resources', 'vlan_networking')
+@tier3
+def test_positive_update_default_org(
+        session, module_org, module_loc, provisioning_env):
+    """Change the default org of more than one discovered hosts
+    from 'Select Action' drop down
+
+    :id: fe6ab6e0-c942-46c1-8ae2-4f4caf00e0d8
+
+    :Setup: Host should already be discovered
+
+    :expectedresults: Default org should be successfully changed for
+        multiple hosts
+
+    :CaseLevel: System
+    """
+    new_org = entities.Organization().create()
+    module_loc.organization.append(new_org)
+    module_loc.update(['organization'])
+    with LibvirtGuest() as pxe_1_host, LibvirtGuest() as pxe_2_host:
+        host_names = [pxe_1_host.guest_name, pxe_2_host.guest_name]
+        with session:
+            session.organization.select(org_name=module_org.name)
+            session.location.select(loc_name=module_loc.name)
+            for host_name in host_names:
+                host_values = session.discoveredhosts.wait_for_entity(
+                    host_name)
+                assert host_values['Name'] == host_name
+            values = session.discoveredhosts.search(
+                'name = "{0}" or name = "{1}"'.format(*host_names)
+            )
+            assert set(host_names) == {value['Name'] for value in values}
+            session.discoveredhosts.assign_organization(
+                host_names,
+                new_org.name
+            )
+            assert not session.discoveredhosts.search(
+                'name = "{0}" or name = "{1}"'.format(*host_names)
+            )
+            session.organization.select(org_name=new_org.name)
+            values = session.discoveredhosts.search(
+                'name = "{0}" or name = "{1}"'.format(*host_names)
+            )
+            assert set(host_names) == {value['Name'] for value in values}
+
+
+@run_in_one_thread
+@run_only_on('sat')
+@skip_if_not_set('compute_resources', 'vlan_networking')
+@tier3
+def test_positive_update_default_location(
```

Comment:
\+ this will as well become t2 if fake host is used.

Reply:
@ldjebran - i would simply generate a new uuid so it gets properly synced

Reply:
@rplevka after some experiments to join this 2 test come to that this will very complicate the tests that became unreadable , and hard to maintain , prefer to let them as is.

Reply:
now even if I want to do it I can't because of the bug, the test as is now , was tested to work.